### DISCUSSION TITLE
Logs location

### DISCUSSION THREAD

**Discussion by Slejpnir** [mKeRix/room-assistant#626] [Q&A] [answered]
Hello,
Sorry for such simple question, but where can I find room-assistant logs after default installation on Pi Zero?

**Comment by mKeRix** [comment, marked as answer]
If you just followed the quickstart guide you setup a systemd service of room-assistant. You can look at the logs of that by typing in `journalctl -u room-assistant.service`.

**Reply by Slejpnir** [reply]
Thanks!
